plans/dht/utils/create-dht-node.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/ipfs/testground/sdk/runtime"
+
+	datastore "github.com/ipfs/go-datastore"
+	libp2p "github.com/libp2p/go-libp2p"
+	host "github.com/libp2p/go-libp2p-core/host"
```

Comment:
You can drop the import prefixes if there's no ambiguity.

Reply:
Happy to follow whatever we use as conventions for Go code and I see that your suggestion was already the convention https://github.com/ipfs/community/blob/master/CONTRIBUTING_GO.md#import-format. Thank you!

That said, I've one question (although I'm not looking to start a bikeshed, as I've said, I'll change it for now :)): Why force the developer to read imports as a zigzag? Or why not prevent future refactors of multiple lines of code in case of a repo name change or just refactor, e.g.:

Today it is:

```
host "github.com/libp2p/go-libp2p-core/host"
```

Next week we could refactor to have

```
host "github.com/libp2p/go-libp2p-host"
```

And with the named import, a dev would only have to change one line rather than go change everywhere to libp2phost given the package name conventions.